### PR TITLE
chore(graphql api): Standardize metric names

### PR DESCRIPTION
**Pull request by lucperkins**
This PR updates the metric names exposed via the GraphQL API to be in line with their Prometheus equivalents (and consistent with [Prometheus naming conventions](https://prometheus.io/docs/practices/naming/)):

* `bytesProcessedTotal` => `processedBytesTotal` (in line with `processed_bytes_total`)
* `eventsProcessedTotal` => `processedEventsTotal` (in line with `processed_events_total`)

I also took the liberty of updating underlying code and filenames to reflect this, although that may have been overreach. If so, I'm happy to revert and change only user-facing names.

### CODE REVIEW

File: src/api/schema/metrics/mod.rs [hunk 1]
```diff
@@ -108,30 +108,30 @@ impl MetricsSubscription {
...
-    ) -> impl Stream<Item = BytesProcessedTotal> {
+    ) -> impl Stream<Item = ProcessedBytesTotal> {
         get_metrics(interval).filter_map(|m| match m.name.as_str() {
-            "processed_bytes_total" => Some(BytesProcessedTotal::new(m)),
+            "processed_bytes_total" => Some(ProcessedBytesTotal::new(m)),
             _ => None,
         })
     }
 
     /// Bytes processed throughput, sampled over a provided millisecond `interval`.
```
Comment (leebenson) [inline src/api/schema/metrics/mod.rs:133]: I might also change the comments to reflect "Processed bytes" vs. "Bytes processed"

**Review by leebenson — APPROVED** [review]
LGTM 👍 

I left a minor note about switching commentary to the "Processed <x>" format, so this translates to public API documentation, but this PR looks great otherwise.

cc @sghall for front-end changes.

**Comment by lucperkins** [comment]
@leebenson I went back and updated those descriptions and took some liberties to make them sound a bit more human. Happy to continue refining if needed!

**Comment by sghall** [comment]
@lucperkins this looks good! Yeah, my OCD was kinda flaring up with these small inconsistencies.  Should sleep better tonight. 🛌    I'll get this updated in the UI once this merges.

**Pull request merged**